interpreter/cling/lib/Interpreter/TabCompletion.h

```diff
@@ -0,0 +1,43 @@
+//--------------------------------------------------------------------*- C++ -*-
+// CLING - the C++ LLVM-based InterpreterG :)
+// author:  Bianca-Cristina Cristescu <bianca-cristina.cristescu@cern.ch>
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+//------------------------------------------------------------------------------
+
+#ifndef CLING_TAB_COMPLETION
+#define CLING_TAB_COMPLETION
+
+#include "textinput/Callbacks.h"
```

Comment:
That's "bad" - lib/Interpreter should not depend on textinput... Do we really need this? Is this object ever registered with textinput?